apps/web/src/components/Lessons/ManageLesson.tsx

```diff
@@ -143,6 +158,30 @@ const ManageLesson = ({ close, tutorId, ...payload }: Props) => {
     [payload, createLessonMutation, tutorId, updateLessonMutation]
   );
 
+  const onSendVerifyEmailSuccess = useCallback(() => {
+    toast.success({
+      title: intl("student-settings.confirm-email.success.title"),
+      description: intl("student-settings.confirm-email.success.desc"),
+    });
```

Comment:
As the name implies, these ids shold be used in the student settings page!. If you want to re-use it here. Move it under a common namespace and then update all palces that reference the old ids. 


```suggestion
    toast.success({
      title: intl("send-confirm-email.success.title"),
      description: intl("send-confirm-email.success.desc"),
    });
```